app/javascript/packages/document-capture/components/in-person-call-to-action.spec.tsx

```diff
@@ -1,18 +1,15 @@
 import sinon from 'sinon';
+import { computeAccessibleName } from 'dom-accessibility-api';
```

Comment:
just curious about what this does

Reply:
My superficial understanding is that this library [evaluates the dom using w3c accessibility standards](https://w3c.github.io/accname/) to emulate the computed accessible values (name, description) of compliant user agents.